lib/test_cleanup.rb

```diff
@@ -46,9 +46,11 @@ def delete_or_move(paths)
     def cleanup_shared
       FileUtils.chmod_R("u+X", HOMEBREW_CELLAR, force: true)
 
-      cleanup_git_meta(repository)
-      clean_if_needed(repository)
-      prune_if_needed(repository)
+      if repository.exist?
+        cleanup_git_meta(repository)
```

Comment:
IMO these should be handled in the methods themselves but: fine with me, no need to change unless you agree. Thanks for the fix!